extensions/wikia/TemplateClassification/TemplateClassification.i18n.php

```diff
@@ -34,9 +34,11 @@
 	'template-classification-edit-modal-cancel-button-text' => 'Cancel',
 	'template-classification-edit-modal-title-edit-type' => 'Edit template type',
 	'template-classification-edit-modal-title-select-type' => 'Choose template type',
+	'template-classification-edit-modal-title-bulk-types' => 'Bulk edit template type',
 	'template-classification-edit-modal-help' => 'To learn more about each template type, including examples, please visit [[Help:Template Types]].',
 	'template-classification-edit-modal-success' => 'New template type has been saved. Thank you!',
 	'template-classification-edit-modal-error' => 'Unfortunately we were not able to save this template type. Would you mind trying again?',
+	'template-classification-edit-modal-bulk-error' => 'Unfortunately we were not able to classify this pages: $1. Would you mind trying again?',
```

Comment:
Another edge case to consider: what if every page fails due to say the service being unavailable, should we list every page even if there are 100?